modules/electrophysiology_browser/php/electrophysiology_browser.class.inc

```diff
@@ -0,0 +1,148 @@
+<?php declare(strict_types=1);
+
+/**
+ * This class features the code for the menu portion of the LORIS
+ * electrophysiology browser module.
+ *
+ * PHP version 7
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Electrophysiology
+ * @author     Loris team <info-loris.mni@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://github.com/aces/Loris
+ */
+namespace LORIS\electrophysiology_browser;
+/**
+ * Provides the PHP code for the menu filter for the electrophysiology browser
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Electrophysiology
+ * @author     Loris team <info-loris.mni@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://github.com/aces/Loris
+ */
+
+class Electrophysiology_Browser extends \DataFrameworkMenu
+{
+
+    public $AjaxModule   = true; // stops from running the query twice
+    public $skipTemplate = true; // stops from looking for a smarty template
```

Comment:
both of these are done by the DataFrameworkMenu constructor, these lines should just be deleted